File: spec/code_samples/C#/user-sol@{ruc}@{user}/get.cs
using System.Net;
using Newtonsoft.Json;

var ruc  = "20000000001";
var user = "MODDATOS";
var token = "YOUR-TOKEN";

using (WebClient wc = new WebClient())
{
   var json = wc.DownloadString($"http://consulta.pe/api/v1/user-sol/{ruc}/{user}?token={token}");
   var isValid = JObject.Parse(json);
}

File: spec/code_samples/PHP/user-sol@{ruc}@{user}/get.php
$ruc  = '20000000001';
$user = 'MODDATOS';
$token = 'YOUR-TOKEN';

$json = file_get_contents("http://consulta.pe/api/v1/user-sol/$ruc/$user?token=$token");

$isValid = json_decode($json);
